CHANGELOG.md

```diff
@@ -1,5 +1,67 @@
 # Azure Kubernetes Service Changelog
 
+## Release 2025-04-27
+
+Monitor the release status by region at [AKS-Release-Tracker](https://releases.aks.azure.com/). This release is titled `v20250427`.
+
+### Announcements
+* AKS supported Kubernetes version release updates are now available in [AKS Release Tracker](https://releases.aks.azure.com/). You can check current in-support Kubernetes versions and LTS versions for specific region and track new patches version release progress with Release Tracker.
+* Kubernetes 1.29 community version is now deprecated.
+* Kubernetes 1.28 and 1.29 are now recognized as Long-Term Support (LTS) releases in AKS, joining existing LTS versions 1.27 and 1.30. For more information, see [Long Term Support (LTS)](https://learn.microsoft.com/azure/aks/long-term-support).
```

Comment:
1.28, 1.29 LTS is old news. Please change this to K8s versions 1.31 and 1.32 are now LTS compatible. Also add that you can view when these LTS releases hit your region in real time via the Release tracker.

Reply:
Incorporated